qa/tasks/salt_manager.py

```diff
@@ -227,6 +227,21 @@ def start_minions(self):
         """Starts salt-minion.service on all cluster nodes"""
         self.__systemctl_cluster(subcommand="start", service="salt-minion")
 
+    def enable_minions(self):
+        """Enables salt-minion.service on all cluster nodes"""
+        self.__systemctl_cluster(subcommand="enable", service="salt-minion")
+
+    def enable_masters(self):
+        """Enables salt-master.service on all cluster nodes"""
+        self.__systemctl_cluster(subcommand="enable", service="salt-master")
+
+    # FIXME: run in a try-wait loop, because it sometimes does this:
+    # Running: "sudo sh -c 'salt \\* saltutil.sync_all 2>/dev/null'"
+    # stdout:target192168000035.teuthology:
+    # stdout:    Minion did not return. [No response]
+    # (which means capturing the stdout and parsing it to ensure that
+    # the string "Minion did not return" did not appear after any of
+    # the minion names)
```

Comment:
Please drop the comment block, above (it got pulled in inadvertently in your recent rebase, I assume).